spec/support/helpers.rb

```diff
@@ -1,5 +1,18 @@
 module Spec
   module Helpers
+    def set_temp_config(config)
```

Comment:
I think renaming this to `with_config` would match with Ruby conventions for methods that take a block and then unset after.